modules/ROOT/partials/protocols.adoc

```diff
@@ -0,0 +1,29 @@
+// Type: reference
+[id='protocols-{context}']
+= Supported standards and protocols
+
+{skupper-name} supports the following protocols for your {service-network}:
+
+* TCP - default
+* HTTP1
+* HTTP2
+
+When exposing or creating a service, you can specify the protocol, for example:
+
+[source,bash,options="nowrap"]
+----
+$ skupper expose deployment hello-world-backend --port 8080 --protocol <protocol> 
+----
+
+where `<protocol>` can be:
+
+* tcp
+* http
+* http2
+
+
+When choosing which protocol to specify, note the following:
+
+* HTTP1 and HTTP2 work when you specify `tcp`. Just because a service is a web service does not mean you must specify a HTTP variant. 
```

Comment:
For user consideration, the primary difference is tcp is balanced at the connection level while http is balanced per request.

tcp - single streamed message, balanced across destination services at connection setup time
http - request/response message routing, balanced across destinations services per request

Reply:
I've updated, but trying to avoid 'balanced', unless you think we must mention that?